code/modules/reagents/Chemistry-Goon-Toxins.dm

```diff
@@ -204,4 +204,163 @@ datum/reagent/itching_powder/on_mob_life(var/mob/living/M as mob)
+				M.adjustBruteLoss(100) // rip in pepperoni
+	..()
+	return
+
+datum/reagent/pancuronium
+	name = "Pancuronium"
+	id = "pancuronium"
+	description = "10 cycles to paralysis, 7% chance to cause 3-5 LOSEBREATH."
+	reagent_state = LIQUID
+	color = "#CF3600" // rgb: 207, 54, 0
+	metabolization_rate = 0.2
+
+datum/reagent/pancuronium/on_mob_life(var/mob/living/M as mob)
+	if(!M) M = holder.my_atom
+	if(current_cycle >= 10)
+		M.SetParalysis(3)
+	if(prob(7))
+		M.losebreath += rand(3,5)
+	..()
+	return
+
+datum/reagent/sodium_thiopental
+	name = "Sodium Thiopental"
+	id = "sodium_thiopental"
+	description = "KOs in ten cycles. Major total stamina penalty."
+	reagent_state = LIQUID
+	color = "#CF3600" // rgb: 207, 54, 0
+	metabolization_rate = 0.7
+
+datum/reagent/sodium_thiopental/on_mob_life(var/mob/living/M as mob)
+	if(!M) M = holder.my_atom
+	if(current_cycle >= 10)
+		M.sleeping += 3
+	M.adjustStaminaLoss(10)
+	..()
+	return
+
+datum/reagent/sulfonal
+	name = "Sulfonal"
+	id = "sulfonal"
+	description = "+1 TOX, KOs in 22 cycles."
+	reagent_state = LIQUID
+	color = "#CF3600" // rgb: 207, 54, 0
+	metabolization_rate = 0.1
+
+/datum/chemical_reaction/sulfonal
+	name = "sulfonal"
+	id = "sulfonal"
+	result = "sulfonal"
+	required_reagents = list("acetone" = 1, "diethylamine" = 1, "sulfur" = 1)
+	result_amount = 3
+
+datum/reagent/sulfonal/on_mob_life(var/mob/living/M as mob)
+	if(!M) M = holder.my_atom
+	if(current_cycle >= 22)
+		M.sleeping += 3
+	M.adjustToxLoss(1)
+	..()
+	return
+
+datum/reagent/amanitin
+	name = "Amanitin"
+	id = "amanitin"
+	description = "On the last cycle that it's in you, it hits you with a stack of TOX damage based on elapsed cycles * rand(2,4). The more you use, the longer it takes before anything happens, but the harder it hits when it does."
+	reagent_state = LIQUID
+	color = "#CF3600" // rgb: 207, 54, 0
+
+datum/reagent/amanitin/on_mob_life(var/mob/living/M as mob)
+	if(!M) M = holder.my_atom
+	..()
+	return
+
+datum/reagent/amanitin/reagent_deleted(var/mob/living/M as mob)
+	if(!M) M = holder.my_atom
+	M.adjustToxLoss(current_cycle*rand(2,4))
+	..()
+	return
+
+datum/reagent/lipolicide
+	name = "Lipolicide"
+	id = "lipolicide"
+	description = "+1 TOX unless they keep eating food."
+	reagent_state = LIQUID
+	color = "#CF3600" // rgb: 207, 54, 0
+
+/datum/chemical_reaction/lipolicide
+	name = "lipolicide"
+	id = "lipolicide"
+	result = "lipolicide"
+	required_reagents = list("mercury" = 1, "diethylamine" = 1, "ephedrine" = 1)
+	result_amount = 3
+
+datum/reagent/lipolicide/on_mob_life(var/mob/living/M as mob)
+	if(!M) M = holder.my_atom
+	if(!holder.has_reagent("nutriment"))
+		M.adjustToxLoss(1)
+	M.nutrition -= 10 * REAGENTS_METABOLISM
+	M.overeatduration = 0
+	if(M.nutrition < 0)//Prevent from going into negatives.
+		M.nutrition = 0
+	..()
+	return
+
+datum/reagent/coniine
+	name = "Coniine"
+	id = "coniine"
+	description = "+2 TOX, +5 LOSEBREATH."
```

Comment:
Sorry to fill up the line-by-line comments and stuff, but again, this is a goon description that can be seen IC